src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -14,12 +14,15 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Python
 {
     [BuildProperty(
         VirtualEnvironmentNamePropertyKey, "If provided, will create a virtual environment with the given name.")]
+    [BuildProperty(
+        ZipVenvDirPropertyKey, "If provided, the virtual environment folder will be zipped to the output folder.")]
```

Comment:
`If provided` => `if true`?

Reply:
it doesn't need to be true, just to be present